pkgs/servers/nextcloud/documentation.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  python3,
+  version,
+  hash,
+}:
+stdenvNoCC.mkDerivation {
+  name = "nextcloud-documentation";
+
+  src = fetchFromGitHub {
+    inherit hash;
+    owner = "nextcloud";
+    repo = "documentation";
+    rev = "refs/tags/v${version}";
+    fetchSubmodules = true;
```

```suggestion
    owner = "nextcloud";
    repo = "documentation";
    tag = "v${version}";
    inherit hash;
```

There are not submodules in https://github.com/nextcloud/documentation